original/zip_image_search.py

```diff
@@ -0,0 +1,178 @@
+##Intro code--import libraries and modules, load face detection classifier, import the small file to with which to practice,
+##and then open the zip file and create a list of the files inside
+
+
+#     file_name_str = 'readonly/small_img.zip'
+#     face_cascade_str = 'readonly/haarcascade_frontalface_default.xml'
+
+def main (name_str, file_name_str, face_cascade_str):
+    
+    import zipfile
+
+    from PIL import Image
+    from PIL import ImageDraw
+    import pytesseract
+    import cv2 as cv
+    import numpy as np
+
+    file_name_str = file_name_str
+    face_cascade_str = face_cascade_str
+    
+    search_name_return_faces (name_str, file_name_str, face_cascade_str)
+
+#this function accepts a zipfile as a string and returns a list of the files, in this case, image files
```

Comment:
I love this comment explaining the function. Its very helpful, you can make it multiple lines as well to improve readability.